### DISCUSSION TITLE
Synchronizing to Backblaze B2 with Cryptomator Vault -- why aren't the original modified times crossing over?

### DISCUSSION THREAD

**Discussion by flavorgold1** [iterate-ch/cyberduck#14451] [Q&A]
So, I am interested in synchronizing over some folders to Backblaze B2 inside a Cryptomator Vault.

The problem is that none of the original 'modified times' are crossing over properly.

Is there some setting that I'm unaware of?

**Comment by dkocher** [comment]
This is supposed to work as we set the modification date in `src_last_modified_millis` metadata for the uploaded file [^1] . I couldn't reproduce this just now testing with uploading a file to a vault on B2 using Cyberduck.

[^1]: 49177c8f

**Comment by dkocher** [comment]
I can reproduce the problem when uploading a file using _Online_ connect mode in Mountain Duck. The problem here is that, the metadata of the file cannot be modified after initiating the upload [^1]. This is not an issue with _Smart Synchronization_ mode where the upload is scheduled to be executed in the background only after we receive the correct modification date from the application writing the file.

[^1]: https://www.backblaze.com/b2/docs/b2_upload_file.html

**Comment by flavorgold1** [comment]
I wonder if there's a difference between the Windows version and the macOS version, on this issue.

**Reply by dkocher** [reply]
The issue will be reproducible on macOS in most cases unless an application sets the modification date on the file prior closing the file after write.
